packages/devtools_app/lib/src/shared/charts/flame_chart.dart

```diff
@@ -455,10 +455,11 @@ abstract class FlameChartState<T extends FlameChart,
   KeyEventResult _handleKeyEvent(RawKeyEvent event) {
     // Only handle down events so logic is not duplicated on key up.
     if (event is RawKeyDownEvent) {
-      // Handle zooming / navigation from W-A-S-D keys.
+      // Handle zooming / navigation from WASD keys or ,AOE keys (Dvorak users)
       // TODO(kenz): zoom in/out faster if key is held. It actually zooms slower
       // if the key is held currently.
-      if (event.logicalKey == LogicalKeyboardKey.keyW) {
+      if (event.logicalKey == LogicalKeyboardKey.keyW ||
+          event.logicalKey == LogicalKeyboardKey.comma) {
```

Comment:
Does this mean that ,OE will also allow for navigation by QWERTY users? If so, we should add a setting that allows for this to be toggled if we're unable to detect keyboard layout.